modules/apps/transfer/types/msgs.go

```diff
@@ -12,6 +12,8 @@ import (
 	ibcerrors "github.com/cosmos/ibc-go/v8/modules/core/errors"
 )
 
+const MaximumMemoLength = 32768 // maximum length of the memo in bytes
```

Comment:
should we document that this was an arbitrary selection?

Reply:
I will do it.